TAInformation/views.py

```diff
@@ -1,12 +1,29 @@
-from django.shortcuts import render, redirect
 from django.http import HttpResponse
-
+from django.shortcuts import render, redirect
 # Create your views here.
 from django.views import View
 
-from TAInformation.models import User
+from TAInformation.Models.account_type import AccountType
+from TAInformation.Models.admin import UserAdmin
+from TAInformation.Models.base_user import BaseUser
+from TAInformation.Models.instructor import Instructor
+from TAInformation.Models.ta import TA
+from TAInformation.models import User, Course
 
 
+def get_user(my_user_id: int):
+    m = User.objects.get(user_id=my_user_id)
+    role = m.role
+    if role == AccountType.DEFAULT.value:
+        return BaseUser(m.user_id, m.name, m.password, m.email, m.home_address, m.phone)
```

Comment:
Does this works?  If so, I might need to place an @abstract_property decorator on the base class constructor.

Reply:
It does work, which surprised me too.

Reply:
I think for now I'll keep it as is, but a change might be made during sprint 2.